<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>word_spell_training</class>
 <widget class="QWidget" name="word_spell_training">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>单词拼写训练</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#word_spell_training{border-image: url(:/Yosemite-Color-Block.png);}</string>
  </property>
  <widget class="QLabel" name="label_en">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 84, 145);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid rgb(30, 84, 145);/*设置边框的粗细，以及颜色*/
border-radius: 15px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
background-color: rgb(238, 192, 218);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>64</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 84, 145);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="but_next">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>下一题</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_jd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 192, 218);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
